### ISSUE TITLE
Empty strings are skipped

### ISSUE THREAD

**Issue by juergba** [astur/arrgv#1]
```js
const arrgv = require('arrgv');
let str = String.raw`one "" two ""`;
let res = arrgv(str);
console.log(res);
```
Output via Git Bash on Windows 10:
- `process.argv`: `[..., ..., 'one', '', 'two', '']`
- `res`:  `['one', 'two']`

**Comment by astur** [comment]
Yes, it is bug. I have same result via Bash on Mac.

Thank you for report.

**Comment by astur** [comment]
I have fixed it in 1.0.2.

**Comment by juergba** [comment]
Thanks you!
